baselines/flanders/flanders/conf/strategy/flanders.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: flanders
+
+strategy:
+  _target_: flanders.strategy.Flanders
+  to_keep: 1  # number of benign local models to keep for the aggregation (FedAvg)
```

Comment:
@jafermarq Sorry to bother you, but I can't wrap my head around this: how can I access a variable of this file from `main.py`?

Reply:
It should be something like:
```bash
python -m flanders.main strategy.strategy.to_keep=2
```
The first `strategy` is because that's the name of the "node" in `defaults`. The second one is to access the `strategy` node in `flanders.yaml`. 

Does it work?

Reply:
Works. Thanks.